root/tree/index/runindex64.C

```diff
@@ -5,6 +5,12 @@
 bool test(TTree*);
 
 const char* fname = "index64.root";
+  // Apple M1 has long double == double; these values exceed its range
```

Comment:
On the github output, the alignment seems off (2 spaces instread of zero).

Reply:
Thanks fixed!